quupe-RN/js/screens/SignIn/SignInContainer.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
```

Comment:
remove text and view import since you are not using it. Also, add ESLint to help you write better code.